sklearn/svm/_classes.py

```diff
@@ -76,6 +76,9 @@ class LinearSVC(LinearClassifierMixin, SparseCoefMixin, BaseEstimator):
         chosen `loss`, `multi_class` and `penalty`, then dual will be set to True,
         otherwise it will be set to False.
 
+        .. versionadded:: 1.3
+           The option `"auto"` is added.
+
         .. versionchanged:: 1.3
            The default value will change from `True` to `"auto"` in 1.5.
```

Comment:
I don't think it needs a `versionadded`, we could just modify this text to be:

The `"auto"` option is added in version 1.3 and will be the default instead of `True` in 1.5.